packages/wrangler/src/__tests__/cloudchamber/apply.test.ts

```diff
@@ -91,7 +104,17 @@ describe("cloudchamber apply", () => {
 
 	mockAccountId();
 	mockApiToken();
-	beforeEach(mockAccount);
+	beforeEach(() => {
+		mockAccount({
+			external_account_id: "test_account_id",
```

Comment:
Use `some-account-id` which is the standard mocked account ID used elsewhere (for some reason a lot of our tests were using `test_account_id` but I've been trying to phase that out).

```suggestion
			external_account_id: "some-account-id",
```